openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -288,7 +288,7 @@ def linkedin_add_to_profile_url(self):
                 get_certificate_url(course_id=self.course_key, uuid=user_certificate.verify_uuid)
             )
             return linkedin_config.add_to_profile_url(
-                self.course_overview.display_name, user_certificate.mode, cert_url, certificate=user_certificate,
+                self.overview, user_certificate.mode, cert_url, certificate=user_certificate,
```

Comment:
The tests are failing, because calling `self.overview` in this place will raise an error - `overview` is a string that contains overview HTML content for the course. You must pass a `CourseOverview` obejct to `add_to_profile_url`, which can be accessed via `self.course_overview`.
```suggestion
                self.course_overview, user_certificate.mode, cert_url, certificate=user_certificate,
```

Reply:
Yes, you are correct. This is copy paste error  as I am working in TEAK branch in order to verify my code and then copy my changes to the branch this PR is associated with.

Thank you for the support!